client/controllers/incidentReports/incidentTable.coffee

```diff
@@ -0,0 +1,35 @@
+Template.incidentTable.onCreated ->
+  @rowsSelected = new ReactiveVar(false)
+
+selectedIncidents = (event, instance) ->
+  # return all the selected incident-ids
+  $('table.incident-table tr.active td.count').map(() ->
+    $(this).data("incident-id")
+  ).get()
+
+changeIncidentStatus = (status) ->
+  incidentIds = selectedIncidents()
+  $.each incidentIds, (index, incidentId) ->
+    Meteor.call 'getIncidentReport', incidentId, (error, incident) ->
```

Comment:
I wonder if this method call is necessary. Could we update the IR and only set the accepted property?